src/crawlers.ts

```diff
@@ -130,6 +131,31 @@ export async function createAndStartContentCrawler(
     return { key, crawler };
 }
 
+/**
+ * PreNavigation hook that blocks resources based on the blockMedia setting
+ * from the request's userData.
+ * Only blocks resources if blockMedia is true.
+ */
+async function blockMediaResourcesHook({ page, request }: PlaywrightCrawlingContext<ContentCrawlerUserData>) {
+    await page.route('**/*', async (route) => {
```

Comment:
`page.route` disables native browser cache which is why `blockRequests` is normally recommended (that is a native Chromium CDP call). The cache disabling is only bad if you do more requests for the same site. I would do a perf test on more URLs of the same site and test more sites because this could slow us down as well.